serverless/serverless
# Update Azure documentation.

## What did you implement:

Better documentation for both getting started and working with serverless on Azure.



## How did you implement it:

Ran through each step with all of the latest tools and made sure everything was up to date.


## How can we verify it:

Check out the diff. You'll see some updated authentication steps (using the latest CLI) and some spelling corrections.



## Todos:

- [x] Write documentation :trollface: 

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
